cmake/Modules/CopyMSVCBins.cmake

```diff
@@ -179,57 +179,61 @@ file(GLOB RNNOISE_BIN_FILES
      "${RNNOISE_INCLUDE_DIR}/../bin${_bin_suffix}/rnnoise*.dll"
      "${RNNOISE_INCLUDE_DIR}/../bin/rnnoise*.dll")
 
+set(QtCore_DIR "${Qt${QT_VERSION}Core_DIR}")
+cmake_path(SET QtCore_DIR_NORM NORMALIZE "${QtCore_DIR}/../../..")
```

Comment:
Is `cmake_path` ensured to always resolve a path with a final forward slash?

Reply:
In my testing on Windows, it appeared to always end with a final forward slash, although that isn't reflected in the documentation. If we want to be really sure that there isn't one, we can try to strip slashes off the right side, but it might be annoying to do so.

Reply:
If the concern about the slashes is blocking, I can split the cosmetic changes into a separate PR for further examination.

Reply:
Nah, fine be me. If it breaks, we'll fix it.